doc/vision.md

```diff
@@ -1,61 +1,68 @@
 
 This library creates self sovereign id's in relation to needs automaticly and stores key information at a discipl-core supported platform (probably some local or private distributed wallet as you want to keep this private)
 
 Enables possibillity to escalate towards an escalation process (convergent facilitation) if any actor does not agree with the automated process and logic or decisions of other actors.
 
-NOTICE: highly experimental - expect the content below to change considerably
+**NOTICE:** highly experimental - expect the content below to change considerably
 
 ------------------------
 
-discipl-law-reg includes a specification for a FLINT model expressed as linked data. For now it is published as verfiable claim with the whole model as JSON-LD as object. The subject of this claim can be used to verify whether the model is official. The model is closely tied to official law and regulations using official jurisdictional references. A FLINT model is created by hand though with a formal method called Calculemus and specific tooling this can be made straightforward and possibly automated to a certain extend (ongoing research).
+## FLINT models
+discipl-law-reg includes a specification for a FLINT model expressed as linked data. For now it is published as a verfiable claim with the whole model as JSON-LD as object. The subject of this claim can be used to verify whether the model is official. The model is closely tied to official law and regulations using official jurisdictional references. A FLINT model is created by hand though with a formal method called _Calculemus_ and specific tooling. This can be made straightforward and possibly automated to a certain extend (ongoing research).
 
-A flint model describes, closely related to official law texts in natural language, the actions actors can or must take in the future (actions and duties) under what conditions given a current situation which depends on the actions that have been taken before and evaluated facts.
+A flint model describes, closely related to official law texts in natural language, the actions actors can or must take in the future (actions and duties). It also describes under what conditions, given a current situation which depends on the actions that have been taken before and evaluated facts.
 
-A flint-ld model within discipl consists out of 4 types of datastructures:
+A flint-ld model within discipl consists out of 4 types of datastructures: model, act, fact and duty.
 
-1) model : datastructure that defines a process closely tied to official law and regulation texts, and which is expressed as related actions in which an actor attends to a need for an object expressed by another actor as interested party and can solve this need by denoting the action as been taken which is only possible if the preconditions are met. It defines at least the following attributes
+### Model
```

Comment:
I would be good with removing some stuff here and linking the document you wrote above. They largely have the same content right?